PUBLISHING.md

```diff
@@ -79,10 +79,11 @@ cargo make revoke-ami -e REVOKE_FROM_USERS=0123456789,9876543210
 
 ## Build a repo
 
+__NOTE: If you intend to replace hosts rather than updating them, you don't need to build an update repository.__
```

Comment:
Can we use the same note format as elsewhere in the doc, e.g. a few lines up?

```suggestion
> NOTE: If you intend to replace hosts rather than update them, you don't need to build an update repository.
```